api/v1alpha7/conversion.go

```diff
@@ -383,6 +383,15 @@ func Convert_v1alpha7_OpenStackClusterSpec_To_v1alpha8_OpenStackClusterSpec(in *
 		}
 	}
 
+	emptySubnet := SubnetFilter{}
```

Comment:
I think it's ok but I'll tag @mdbooth on this one, he might think it's not the right way to do it.